unicoremc/models.py

```diff
@@ -448,6 +469,21 @@ def initiate_create_marathon_app(self, cmd):
                 'Create Marathon app failed with response: %s - %s' %
                 (resp.status_code, resp.json().get('message')))
 
+    def update_marathon_app(self):
+        post_data = self.get_marathon_app_data()
+        app_id = post_data.pop('id')
+        resp = requests.put(
+            '%(host)s/v2/apps/%(id)s?force=true' % {
```

Comment:
I'm a bit uncomfortable with making this a `force=true` can we make that optional?